src/types/index.ts

```diff
@@ -353,7 +362,10 @@ export enum ConditionType {
   IsIdentity = 'IsIdentity',
 }
 
-export type ConditionBase = { target: ConditionTarget; trustedClaimIssuers?: TrustedClaimIssuer[] };
+export type ConditionBase = {
+  target: ConditionTarget;
+  trustedClaimIssuers?: InputTrustedClaimIssuer[];
```

Comment:
We're facing the same problem here, since conditions are also returned to users. Leave it as `TrustedClaimIssuer` for now. I'm working on the `Condition` side

Reply:
I switched it back for now, but I wouldn't want to merge this in like this since its not enough to close the ticket.